Controller/Artist.cs

```diff
@@ -23,6 +23,11 @@ public Model.Artist GetArtist(int artistID)
             Model.Artist artist = GetArtistFromDB(artistID);
             return artist;
         }
+
+        public List<Model.Track> GetArtistTracks(int artistID)
+        {
+            return GetArtistTracksFromDB(artistID);
+        }
```

Comment:
hier zou ik nog een comment plaatsen wat het doet

Reply:
Ik heb deze functie aangepast. Naar 1 functie die GetArtistTracks heet. om de Track en Artist qua format hetzelfde te houden  Staat in de AudioAfspelen Branch.